tests/layer_tests/common/utils/common_utils.py

```diff
@@ -98,6 +98,14 @@ def allclose(cur_array, ref_array, atol, rtol):
     :param rtol: relative tolerance (threshold for relative difference)
     :return: bool value means that values of tensors are equal with tolerance or not
     """
+    try:
+        # a scalar case of string type
+        # need to repack TF output to python string
+        if isinstance(ref_array, bytes):
+            ref_array = np.array(str(ref_array, 'utf-8'))
+    except:
+        pass
```

Comment:
Why `try except` is needed here?

Reply:
that's because not all bytes encode utf-8 string.